### ISSUE TITLE
'Combine selected notes'--how?

### ISSUE THREAD

**Issue by PacoH** [JackGruber/joplin-plugin-combine-notes#8]
I installed the plugin but 'Tools' menu item just says 'Combine selected notes'. There is no visible way to select more than one note, let alone notebooks. I thought this could combine multiple notebooks.

**Comment by JackGruber** [comment]
>I thought this could combine multiple notebooks

No this plugin is not for combining multiple notebooks, only notes. 
https://github.com/JackGruber/joplin-plugin-combine-notes#usage

**Comment by PacoH** [comment]
OK, but how to select more than one note is not at all intuitive. I figured it out but the interface doens't help much.

I see the `Combine selected notes` item in the context menu but I'm not sure what it does. Does it delete the notes that are combined or what? I ended up just dragging and dropping the selected notes to another notebook.

**Comment by JackGruber** [comment]
>OK, but how to select more than one note is not at all intuitive.

Select multiple notes with `Ctrl` and clicken on the note or `Ctrl` + `A` to select all notes. 

> I see the Combine selected notes item in the context menu but I'm not sure what it does

The plugin combines two or more notes into one (it copies the note content into a new note), the original notes are deleted or kept according to the settings.

**Comment by PacoH** [comment]
Oh. I see that there is a setting 'Delete combined notes' which is unchecked. I tested this on 3 selected notes. When I clicked 'Combine selected notes', it created a note named 'Combined note' which, to my surprise, contained the contents of all three notes. I expected to be able to copy the notes to a new notebook. But I see now that the description for the plugin states "Plugin to combine one or more notes to a new one."
